### PR TITLE
[11.0][FWD] add generic exporter extracted from Magento connector

### PR DESCRIPTION
**Pull request by rousseldenis**
Forward-port of https://github.com/OCA/connector/commit/7d10c14cabcae9e0fe3b6e1151f87b1b49f2be10

### CODE REVIEW

**Review by guewen — APPROVED** [review]
Thanks!

File: connector/components/synchronizer.py [hunk 1]
```diff
@@ -101,6 +110,310 @@ class Exporter(AbstractComponent):
...
 
 
+class GenericExporter(AbstractComponent):
+    """ Generic Synchronizer for exporting data from Odoo to a backend """
+    _name = 'generic.exporter'
+    _inherit = 'base.exporter'
+    _default_binding_field = None
+
+    def __init__(self, working_context):
+        super(GenericExporter, self).__init__(working_context)
```
Comment (sebalix) [inline connector/components/synchronizer.py:120]:
```suggestion
        super().__init__(working_context)
```

**Review by sebalix — APPROVED** [review]

**Comment by sebalix** [comment]
@rousseldenis just checked the origin of the code (I think the original v10 PR was based on https://github.com/OCA/connector-magento/blame/10.0/connector_magento/components/exporter.py#L121), do you think you can make use of `--author` to bind the commit to the original author (@guewen )?

**Review by lmignon — APPROVED** [review]

**Comment by guewen** [comment]
> do you think you can make use of `--author` to bind the commit to the original author

@rousseldenis could you amend the commit (the one on #337 too) before we merge please?

**Comment by rousseldenis** [comment]
@guewen Done

**Pull request merged**